### PR TITLE
chore/SOF-6804  Re-implement Refinement process for properties using tags

### PR DESCRIPTION
**Pull request by timurbazhirov**

### CODE REVIEW

File: schema/property/raw.json [hunk 1]
```diff
@@ -23,6 +23,31 @@
         },
         "source": {
             "$ref": "source.json"
+        },
+        "exabyteId": {
```
Comment (timurbazhirov) [inline schema/property/raw.json:27]: We will need to clean up https://github.com/Exabyte-io/esse/blob/172188ec99a625170ab0770ab066364e5c9fe078/schema/property/refined.json#L11 and remove the obsolete entries then or update - i.e. refined property is a raw property with `isRefined` tag

File: schema/property/raw.json [hunk 2]
```diff
@@ -23,6 +23,31 @@
...
+            "description": "total number of properties among which this property is the best.",
+            "type": "number"
+        },
+        "systemTags": {
+            "description": "property system tags, marks property system characteristics, values refined or best (could be both)",
+            "type": "array",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "refined",
```
Comment (timurbazhirov) [inline schema/property/raw.json]: `isRefined`, `isBest`

File: schema/in_memory_entity/base.json [hunk 3]
```diff
@@ -19,6 +19,13 @@
         },
         "systemName": {
             "type": "string"
+        },
+        "systemTags": {
```
Comment (timurbazhirov) [inline schema/in_memory_entity/base.json]: This should be added in the web application `entity` instead.

File: schema/property/raw.json (hunk 2 above)
Comment (unsigned6) [reply schema/property/raw.json]: Done

File: schema/in_memory_entity/base.json (hunk 3 above)
Comment (unsigned6) [reply schema/in_memory_entity/base.json]: Done

**Review by unsigned6 — APPROVED** [review]

**Pull request merged**